docs/spfx/viva/design/quick-view-samples.md

```diff
@@ -50,39 +51,39 @@ Design for presenting inventory details from warehouse.
 
 * Sample at [GitHub](https://github.com/pnp/AdaptiveCards-Templates/tree/main/samples/inventory)
 
-:::image type="content" source="../../../images/viva-design/samples/inventory.png" alt-text="Inventory":::
+:::image type="content" source="../../../images/viva-design/samples/inventory.png" alt-text="Screenshot of a sample inventory details card from a warehouse, showing Distribution center # 73's top selling items.":::
```

Comment:
Maybe replace the hash with "number 73's," if that what the hash represetns in this case.